test-util/src/main/scala/io/buoyant/test/Exceptions.scala

```diff
@@ -0,0 +1,10 @@
+package io.buoyant.test
+
+import org.scalatest.Assertions
+
+trait Exceptions extends Assertions {
+
+  def assertThrows[T <: AnyRef: Manifest](f: => Any): Unit = {
+    val _ = intercept[T](f)
+  }
```

Comment:
Consider providing i.e. io.buoyant.test.FunSuite that mixes in scalatest's FunSuite with all of the helpers that we normally use (Awaits, for instance)...  Then we can simply use io.buoyant.test.FunSuite throughout our code.